.jenkins/pytorch/multigpu-test.sh

```diff
@@ -20,7 +20,7 @@ if [ -n "${IN_CIRCLECI}" ]; then
     sudo apt-get install -y --allow-downgrades --allow-change-held-packages libnccl-dev=2.5.6-1+cuda10.1 libnccl2=2.5.6-1+cuda10.1
   fi
 
-  if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda9-cudnn7-py2* ]]; then
+  if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda10.1-cudnn7-py3* ]]; then
```

Comment:
@ezyang this was the replacement you were talking with @seemethere about, I think.

Reply:
Minor nit, also not merge blocking but we could probably change this to

```suggestion
  if [[ "$BUILD_ENVIRONMENT" == *-xenial-cuda*-cudnn* ]]; then
```

This is to avoid having to update this every time we bump cuda version or cudnn version

Reply:
Yes. Be sure to grep for other similar occurrences too.